src/opnsense/mvc/app/controllers/OPNsense/CaptivePortal/forms/dialogZone.xml

```diff
@@ -18,7 +18,10 @@
         <help><![CDATA[Select interface(s) to enable for captive portal.]]></help>
     </field>
     <field>
-        <id>zone.authservers</id>
+        <id>zone.disableRules</id>
```

Comment:
it looks like this landed in another field?

Reply:
Yeah, github seems a bit slow today, applied your suggestion twice ...